lib/robots/dor_repo/gis_assembly/extract_iso19139.rb

```diff
@@ -19,21 +19,21 @@ def perform(druid)
           rootdir = GisRobotSuite.locate_druid_path druid, type: :stage
 
           # See if generation is needed
-          fn = File.join(rootdir, 'metadata', 'geoMetadata.xml')
-          if File.size?(fn)
-            LyberCore::Log.info "extract-iso19139: #{druid} found #{fn}"
+          geo_metadata_filename = File.join(rootdir, 'metadata', 'geoMetadata.xml')
+          if File.size?(geo_metadata_filename)
+            LyberCore::Log.info "extract-iso19139: #{druid} found #{geo_metadata_filename}"
             return
           end
 
           begin
-            fn = GisRobotSuite.locate_esri_metadata "#{rootdir}/temp"
```

Comment:
by all means, let's reuse `fn` variable to refer to different things in the same function.